src/api/app/controllers/webui/request_controller.rb

```diff
@@ -294,7 +294,12 @@ def change_devel_request_dialog
   end
 
   def change_devel_request
-    required_parameters :devel_project, :package, :project
+    unless params[:devel_project]
```

Comment:
Erm, so if this is required why isn't it also in the routes?

Reply:
Because `devel_project` comes from an input is not part of the routes

Reply:
`devel_project` comes from an input, with autocomplete code. I haven't found a trivial way of including the value of an input in a route.

Reply:
Added a `required` attribute to  `devel_project` input. Also checked that without value an exception is thrown.